packages/platforms/delta/x86-64/x86-64-delta-ag7648/onlp/builds/src/module/src/debug.c

```diff
@@ -0,0 +1,45 @@
+#include "x86_64_delta_ag7648_int.h"
```

Comment:
Please remove this file as it provides no value. 
The debug interface has been superceded by the onlp_sysi_debug() entry point.